config/nrfconnect/overlay-sample-defaults.conf

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_NCS_SAMPLES_DEFAULTS=y
+
+CONFIG_CPLUSPLUS=y
```

Comment:
remove as it's inferred from CONFIG_CHIP=y

Reply:
Done